index.html

```diff
@@ -67,27 +67,27 @@ <h1 class="about">
       </div>
```

Comment:
- [ ] On line 7, add a more meaningful title for the page. You currently have the page title as Document.